week3/my-app/src/App.jsx

```diff
@@ -0,0 +1,32 @@
+import React from 'react';
+import './App.css';
+import { Shoes } from './components/Shoes';
+
+export default class App extends React.Component {
+
+  constructor(props) {
+    super(props);
+
+    this.state = {
+      shoes: [
+       {id: 1, name:"Cizme", price:"300", size:"38", color:"negru", description:"piele"},
+       {id: 2, name:"Ghete", price:"230", size:"36", color: "roz"},
+       {id: 3, name:"Pantofi", price:"180", size:"40", color: "albastru", description:"piele"},
+       {id: 4, name:"Botine", price:"200", size:"39", color:"maro"},
+       {id: 5, name:"Sandale", price:"150", size:"41", color: "rosu"},
+      ]
+    }
+  }
+
+  renderShoes = () => {
+    return this.state.shoes.map(shoes => <Shoes key={shoes.id} name={shoes.name} price={shoes.price} size={shoes.size} description={shoes.description} color={shoes.color} />)
```

Comment:
Ai grija la denumirea variabilelor si a componentelor. In cazul de fata functioneaza asa cum e scris, dar exista cazuri in care nu va functiona, exemplu:
```
const shoes = [.....];
this.state = {
   shoes: shoes
};
```
Functia `map` itereaza prin lista si apeleaza functia parametru cu fiecare element, asta inseamna ca :
`elements.map(element => ...);` este mai corect si mai explicit, fara a crea confuzie sau eventuale erori.
In cazul tau, `this.state.shoes.map(shoes => ...)` ar trebui sa fie `this.state.shoes.map(shoe => ...)`
Acelasi lucru si pentru componenta `Shoes`, daca este la plural, ma astept sa fie o componenta care primeste o lista, nu un obiect.